examples/fungible-token/tests/general.rs

```diff
@@ -0,0 +1,75 @@
+use near_sdk_sim::{
+    call, deploy, init_simulator, to_yocto, view, ContractAccount, UserAccount, STORAGE_AMOUNT,
+};
+use std::str::FromStr;
+
+extern crate fungible_token;
+use fungible_token::FungibleTokenContract;
+
+near_sdk_sim::lazy_static::lazy_static! {
+    static ref TOKEN_WASM_BYTES: &'static [u8] = include_bytes!("../res/fungible_token.wasm").as_ref();
+}
```

Comment:
The tricky part here is that I don't know where the artifact of the contract will end up.  Rust puts things in `target/wasm32-unknown-unknown/..`, but the `target` directory could be in the project root or a workspace root.  Perhaps this can be determined when running the procedural macro, but as you point out this could be a future improvement/simplification.  

Another solution is that we can include the type information in the wasm binary so that reading it will generate the proxy. This has the added benefit of allowing for JS objects in NAJ to be generated as well.

Reply:
So my conclusion with this is we should aim to try to create a solution where the type info comes from the wasm file.  This way we get AS support for free because Rust would be generated from the type info.

I think this feature is a big win too for creating the JS wrappers automatically.

Reply:
That sounds great – we can improve this once we have type info stored in the wasm